lambda/index.js

```diff
@@ -11,10 +11,10 @@ const URL = process.env.URL;
 
 exports.handler = function(event, context, callback) {
   const key = event.queryStringParameters.key;
-  const match = key.match(/(\d+)x(\d+)\/(.*)/);
```

Comment:
Hmm, I haven't tested this, but I wonder if you could achieve the same thing with a smaller change:

```javascript
const match = key.match(/(\d+)x(\d+)(\/.*)/)
```

This moves the leading slash to be part of the pattern. I think that's all the split and concatenation are doing, right?